sros2/.coveragerc

```diff
@@ -0,0 +1,4 @@
+[run]
+omit =
+    # omit test directory
+    test/*
```

Comment:
I did this to get accurate coverage numbers. Code in test is usually pretty well covered and skewing the numbers